README.md

```diff
@@ -1,73 +1,189 @@
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Deployment <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://covid-19-webb-app.netlify.app/)
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://www.loom.com/share/1a25120c521241829f0fd1f35d22d950)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up run the following steps.
+
+type cd dirname, after type git clone [https://github.com/Enekwestanley/covid-19.git]
 
 ### Prerequisites
+- Node.js
+- Git
+- Web Browser
+- Vscode(optional)
 
 ### Setup
+  - Clone the repository to your local machine using `git clone type cd dirname, after type git clone [https://github.com/Enekwestanley/covid-19.git]
+`  command.
+
+  - Change directory into project folder `cd covid-19`
 
-### Install
+  - Install all the dependencies using `npm i`
 
 ### Usage
 
+To run the project, execute the following command:
+- Run `npm start`
+  
+- A new browser will open automatically by using this address [http://localhost:3000](https://stately-moonbeam-331ae3.netlify.app/#/) 
+  
+-  The page will reload when you make changes.
+
 ### Run tests
 
-### Deployment
+To run tests, run the following command:
+
+- Run `npm test`
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **STANLEY TOCHUKWU**
 
+- GitHub: [@Enekwestanley](https://github.com/Enekwestanley)
+- Twitter: [@Enekwestanley](https://twitter.com/Tochukwu87)
+- LinkedIn: [@Enekwestanley](https://www.linkedin.com/in/stanley-enekwe-285104230/)
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-## Authors
+<!-- FUTURE FEATURES -->
 
-👤 **Author1**
+## 🔭 Future Features <a name="future-features"></a>
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-👤 **Author2**
+- [ ] **Enhance the My Profile section by adding the "Read more" button.**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
 
-## 🤝 Contributing
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
 
 Contributions, issues, and feature requests are welcome!
 
 Feel free to check the [issues page](../../issues/).
 
-## Show your support
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
```

Comment:
[REQUIRED] Kindly acknowledge Nelson Sakwa as described in the project requirements